frontend/src/scenes/funnels/funnelLogic.ts

```diff
@@ -1198,26 +1198,10 @@ export const funnelLogic = kea<funnelLogicType>({
     }),
 
     listeners: ({ actions, values, props }) => ({
-        loadResultsSuccess: async ({ insight }) => {
-            if (insight.filters?.insight !== InsightType.FUNNELS) {
-                return
-            }
-            // hide all but the first five breakdowns for each step
-            values.steps?.forEach((step) => {
-                values.flattenedStepsByBreakdown
-                    ?.filter((s) => !!s.breakdown)
-                    ?.slice(5)
-                    .forEach((b) => {
-                        actions.setHiddenById({ [getVisibilityIndex(step, b.breakdown_value)]: true })
-                    })
-            })
-        },
```

Comment:
This behavior has been pretty annoying, in that we were changing visibility preferences from under the user. Looks like it was added in #12113, do you remember what was the motivation @alexkim205?

Reply:
Oh LOL meant #5733